packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -389,12 +390,114 @@ export enum LambdaEdgeEventType {
   VIEWER_RESPONSE = "viewer-response",
 }
 
+interface ViewerCertificateBaseCertficateOptions {
```

Comment:
I think ti makes sense to rename this to `ViewerCertificateOptions` and add `aliases` in here, and then instead of `ViewerCertificateAcmCertficateOptions` and `ViewerCertificateIamCertficateOptions` which contain a single required argument, use it as a positional argument for the relevant `ViewerCertificate` factories:

```ts
ViewerCertificate.fromAcmCertificate(cert, { aliases: [ ... ], ... });
```

Reply:
Much better API, thanks. Done